packages/amplication-design-system/src/components/Button/Button.tsx

```diff
@@ -65,7 +74,7 @@ export const Button = ({
           className="amp-button__icon"
         />
       )}
-      {children}
+      {iconPosition === EnumIconPosition.Left && children}
```

Comment:
if Right is the default (to keep the current behavior) then I would expect to see no change when Right is used.
I think the current options are in reverse
Isn't that the correct option here?
```suggestion
      {iconPosition === EnumIconPosition.Right && children}
```

Reply:
if iconPosition === "left" then icon will should be in left and {children} should be in right direction
as line no. 77 is after icon so the icon will be rendered in the left direction and text(children) will be in right direction;

Reply:
you are correct. I thought that the icon is currently on the right but it is on the left... all good!